modules/33acrossIdSystem.js

```diff
@@ -36,6 +37,8 @@ function calculateQueryStringParams(pid, gdprConsentData) {
   const params = {
     pid,
     gdpr: Number(gdprApplies),
+    src: CALLER_NAME,
+    ver: '$prebid.version$'
```

Comment:
Does the PBJS linter require a comma at the end of this line? If so then please add it. Otherwise, it's fine as is.

Reply:
It's not required.. I am seeing occasions where the comma is added and other ones where it's not.